go1.13/entrypoint.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/sh
+
+set -e
+
+if [ -z "${IMPORT}" ]; then
+  IMPORT="${GITHUB_REPOSITORY}"
+fi
+WORKDIR="${GOPATH}/src/github.com/${IMPORT}"
+
+# PROJECT_PATH specifies the subdirectory in the working directory that the Go project is
+if [ -z "${PROJECT_PATH}" ]; then
+  PROJECT_PATH="."
+fi
+
+# Go can only find dependencies if they're under $GOPATH/src.
+# GitHub Actions mounts your repository outside of $GOPATH.
+# So symlink the repository into $GOPATH, and then cd to it.
+mkdir -p "$(dirname "${WORKDIR}")"
+ln -s "${PWD}" "${WORKDIR}"
+cd "${WORKDIR}/${PROJECT_PATH}"
+
+# If a command was specified with `args="..."`, then run it.  Otherwise,
+# look for something useful to run.
+if [ $# -eq 0 ] || [ "$*" = "" ]; then
+  if [ -r Makefile ]; then
+    make
+  else
+    if [ -r go.mod ]; then
+      export GO111MODULE=on
```

Comment:
For `go1.13` I think the default will be `GO111MODULE=auto` so we don't need this anymore:
https://golang.org/doc/go1.13#modules
> The GO111MODULE environment variable continues to default to auto, but the auto setting now activates the module-aware mode of the go command whenever the current working directory contains, or is below a directory containing, a go.mod file — even if the current directory is within GOPATH/src. This change simplifies the migration of existing code within GOPATH/src and the ongoing maintenance of module-aware packages alongside non-module-aware importers.

But I also think it's probably harmless to have it